Makefile

```diff
@@ -92,6 +92,7 @@ clean:
 	-rm -rf node_modules
 	@if [ -d deps/icu ]; then echo deleting deps/icu; rm -rf deps/icu; fi
 	-rm -f test.tap
+	$(MAKE) clean-addons
```

Comment:
You can add these as prerequisites on the targets themselves, e.g. `clean: clean-addons`